crates/pera-swarm-config/src/genesis_config.rs

```diff
@@ -148,6 +150,18 @@ impl ValidatorGenesisConfigBuilder {
         self
     }
 
+    pub fn with_class_groups_key_pair_and_proof(mut self, key_pair: &AuthorityKeyPair) -> Self {
+        // It is safe to unwrap here because the protocol_key_pair is always set before
+        // also the validator can not be built without the class groups key.
```

Comment:
```suggestion
        // It is safe to unwrap here because the protocol_key_pair is always set before.
        // Also, the validator cannot be built without the class groups key.
```